### PR TITLE
fix(github): fixes bug with github token

### PR DESCRIPTION
**Pull request by cujarrett**
## What

- Fixes bug with GitHub personal access token input handling
- Fixes typo in casing of GitHub personal access token message

## Sample local run

```sh
❯ npm ls -g --depth=0
/usr/local/lib
├── semantic-release-cli@0.0.0-development -> /Users/REDACTED/Developer/cli
└── semantic-release@17.3.1


~/Developer/REDACTED add-semantic-release
❯ semantic-release-cli setup --npm-token=REDACTED
? What is your npm registry? https://registry.npmjs.org/
? Provide a GitHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo) REDACTED
? What CI are you using? Circle CI
? What is your CircleCI API token? REDACTED
? Do you want a `config.yml` file with semantic-release setup? No

~/Developer/glimpse add-semantic-release* 33s
```

@gr2m - Please let me know if this looks ok to fix logging the GitHub token to the package.json via the removal of `pkg` in [PR 352](https://github.com/semantic-release/cli/pull/352/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556L90)

### CODE REVIEW

**Comment by gr2m** [comment]
I don't quite understand how this change would affect anything? You add the `pkg` as first argument to the `github` function, but it's not used at all? What am I missing?

**Comment by cujarrett** [comment]
@gr2m - I beleive it's being mistakenly written from here: https://github.com/semantic-release/cli/blob/master/src/index.js#L115. If you think this is not needed feel free to close.

File: src/lib/github.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = async function (info) {
       type: 'input',
       name: 'token',
       message:
-        'Provide a GItHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo)',
+        'Provide a GitHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo)',
```
Comment (gr2m) [inline src/lib/github.js:23]: do you want to make a separate PR for this typo fix?

**Comment by gr2m** [comment]
I think this change makes no difference, so I'm going to close it. Thank you though!

File: src/lib/github.js (hunk 1 above)
Comment (cujarrett) [reply src/lib/github.js:23]: Fixed in [357](https://github.com/semantic-release/cli/pull/357)
